llama_index/query_engine/retriever_query_engine.py

```diff
@@ -48,6 +49,11 @@ def __init__(
 
         super().__init__(callback_manager)
 
+    def _get_prompts_and_modules(self) -> Tuple[PromptDictType, PromptMixinType]:
+        prompts_dict = {}
+        module_dict = {"response_synthesizer": self._response_synthesizer}
+        return prompts_dict, module_dict
+
```

Comment:
similarly we can just set prompts, which sets prompts for this and its constituent classes?